pkg/driver/efs_watch_dog.go

```diff
@@ -233,7 +233,7 @@ func (w *execWatchdog) runLoop(stopCh <-chan struct{}) {
 	for {
 		select {
 		case <-stopCh:
-			klog.Info("stopping...")
+			//klog.Info("stopping...")
```

Comment:
This line spams the test out and makes the build logs unreadable. @wongma7 LMK if you want me to uncomment before merge

Reply:
no let's remove it, I noticed it in a previous PR too. I'll create an issue to track it, it oculd be a real bug although tests  pass despite it